fdbserver/storageserver.actor.cpp

```diff
@@ -1524,9 +1525,9 @@ struct StorageServer : public IStorageMetricsService {
 		LatencySample updateEncryptionLatencySample;
 
 		LatencyBands readLatencyBands;
-		LatencySample mappedRangeSample; // Samples getMappedRange latency
-		LatencySample mappedRangeRemoteSample; // Samples getMappedRange remote subquery latency
-		LatencySample mappedRangeLocalSample; // Samples getMappedRange local subquery latency
+		std::unique_ptr<LatencySample> mappedRangeSample; // Samples getMappedRange latency
```

Comment:
So a ptr rather than an inline LatencySample? Takes less space.

Reply:
Yes. Use the pointer to save some bytes, apparently `LatencySample` is quite large.